api/net/Areas/Admin/Controllers/SeriesController.cs

```diff
@@ -0,0 +1,142 @@
+using System.Net;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http.Extensions;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.Annotations;
+using TNO.API.Areas.Admin.Models.Series;
+using TNO.API.Models;
+using TNO.DAL.Models;
+using TNO.DAL.Services;
+using TNO.Entities;
+using TNO.Entities.Models;
+
+namespace TNO.API.Areas.Admin.Controllers;
+
+/// <summary>
+/// SeriesController class, provides series endpoints for the admin api.
+/// </summary>
+[Authorize]
+[ApiController]
+[Area("admin")]
+[ApiVersion("1.0")]
+[Route("api/v{version:apiVersion}/[area]/series")]
+[Route("api/[area]/series")]
+[Route("v{version:apiVersion}/[area]/series")]
+[Route("[area]/series")]
+[ProducesResponseType(typeof(ErrorResponseModel), (int)HttpStatusCode.Unauthorized)]
+[ProducesResponseType(typeof(ErrorResponseModel), (int)HttpStatusCode.Forbidden)]
+public class SeriesController : ControllerBase
+{
+    #region Variables
+    private readonly ISeriesService _service;
+    #endregion
+
+    #region Constructors
+    /// <summary>
+    /// Creates a new instance of a SeriesController object, initializes with specified parameters.
+    /// </summary>
+    /// <param name="service"></param>
+    public SeriesController(ISeriesService service)
+    {
+        _service = service;
+    }
+    #endregion
+
+    #region Endpoints
+    /// <summary>
+    /// Find a page of series for the specified query filter.
+    /// </summary>
+    /// <returns></returns>
+    [HttpGet("all")]
+    [Produces("application/json")]
+    [ProducesResponseType(typeof(SeriesModel), (int)HttpStatusCode.OK)]
```

Comment:
This should be an array or `IEnumerable` as we don't return a single `SeriesModel`